lib/resolve-value.js

```diff
@@ -111,24 +116,61 @@ var resolveValue = function(decl, map, /*optional*/ignorePseudoScope, /*internal
 		var variableName = variableFallbackSplitPieces[0].trim();
 		var fallback = variableFallbackSplitPieces.length > 1 ? variableFallbackSplitPieces.slice(1).join(',').trim() : undefined;
 
-		(map[variableName] || []).forEach(function(varDeclMapItem) {
-			// Make sure the variable declaration came from the right spot
-			// And if the current matching variable is already important, a new one to replace it has to be important
-			var isRoot = varDeclMapItem.parent.type === 'root' || varDeclMapItem.parent.selectors[0] === ':root';
+		var process = (ignorePseudo, ignorePseudo2) =>
```

Comment:
This looks related to my comments about `ignorePseudo`/`keepPseudo` above, https://github.com/MadLittleMods/postcss-css-variables/pull/126/files#r608857080

Should comment about why 2?